conda-store-server/conda_store_server/registry.py

```diff
@@ -2,192 +2,23 @@
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
 
-import gzip
-import hashlib
-import urllib.parse
-
-from python_docker.registry import Image, Registry
-from sqlalchemy.orm import Session
-from traitlets import Callable, Dict, default
+from traitlets import Callable, Dict
 from traitlets.config import LoggingConfigurable
 
-from conda_store_server._internal import orm, schema, utils
-
 
 class ContainerRegistry(LoggingConfigurable):
     container_registries = Dict(
         {},
-        help="Registries url to upload built container images with callable function to configure registry instance with credentials",
+        help="(deprecated) Registries url to upload built container images with callable function to configure registry instance with credentials",
         config=True,
     )
 
     container_registry_image_name = Callable(
-        help="Image name to assign to docker image pushed for particular registry",
+        help="(deprecated) Image name to assign to docker image pushed for particular registry",
         config=True,
     )
 
-    @default("container_registry_image_name")
-    def _default_container_registry_image_name(self):
-        def _container_registry_image_name(registry: Registry, build: orm.Build):
-            return f"{registry.username}/{build.environment.namespace.name}-{build.environment.name}"
-
-        return _container_registry_image_name
-
     container_registry_image_tag = Callable(
-        help="Image name and tag to assign to docker image pushed for particular registry",
+        help="(deprecated) Image name and tag to assign to docker image pushed for particular registry",
         config=True,
     )
-
-    @default("container_registry_image_tag")
```

Comment:
This implementation is not used since the docker build workflows have been disabled